File: C#/Programming Basics with C#/13. Nested Loops - Exercise/04. Coding/Coding.cs
using System;
using System.Text;

class Coding
{
    static void Main()
    {
        string nToString = Console.ReadLine();
        int n = int.Parse(nToString);
        int length = nToString.Length;
        //string nToString = n.ToString();


        for (int row = length - 1; row >= 0; row--)
        {
            int number = n % 10;
            n = (n - number) / 10;

            if (number != '0')// Тук условието е грешно
            {
                int sum = 33 + number;
                char symbol = Convert.ToChar(sum);
                for (int i = 0; i < number; i++)
                {
                    Console.Write(symbol);
                }
                Console.WriteLine();
            }
            else
            {
                Console.WriteLine("ZERO");
            }
        }
    }
}

File: C++/Programming Basics with C++/13. Exercise - Nested loops/simple_task/Coding.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int n;
    cin >> n;
    string nToString = to_string(n);
    int length = nToString.size();

    for (int row = 0; row < length; row++) {
        int number = nToString[length - 1 - row] - '0';

        if (number == 0) {
            cout << ("ZERO") << endl;
            continue;
        }

        char symbolToRepeat = (char) (number + 33);

        for (int i = 0; i < number; i++) {
            cout << symbolToRepeat;
        }
        cout << endl;
    }

    return 0;
}
